src/com/owncloud/android/ui/adapter/LocalFileListAdapter.java

```diff
@@ -157,7 +166,9 @@ public View getView(int position, View convertView, ViewGroup parent) {
                 		        );
                             fileIcon.setImageDrawable(asyncDrawable);
                             task.execute(file);
-                        }
+                            Log_OC.v(TAG, "Executing task to generate a new thumbnail");
```

Comment:
Well, depends on what you really want to log. In this case I wanted to have a look on how many tasks were requested and how many were cancelled. Where to put the log in the Task? In doInBackground? Maybe it's not executed if something fails in preExecute(), and if it runs may be "quite later" than the call to 'execute'. 

But I see your point, having logs in the main methods on ```AsyncTasks``` would help in a general case for debugging.